### PR TITLE
Fix CircleCi 32-bit Python 2.7 build for v2.0.x

### PR DESCRIPTION
**Pull request by astrofrog**
Not sure if this will work, but worth a try!

### CODE REVIEW

**Comment by astrofrog** [comment]
It worked! 🎉

File: .circleci/config.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ jobs:
           command: /opt/python/cp27-cp27m/bin/pip install numpy scipy "pytest<3.7" pytest-astropy pytest-xdist Cython jinja2
       - run:
           name: Run tests
-          command: PYTHONHASHSEED=42 /opt/python/cp27-cp27m/bin/python setup.py test --parallel=4 || (git reset HEAD && PYTHONHASHSEED=42 /opt/python/cp27-cp27m/bin/python setup.py test --parallel=4)
+          command: PYTHONHASHSEED=42 /opt/python/cp27-cp27m/bin/python setup.py test --offline --no-git --parallel=4 || (git reset HEAD && PYTHONHASHSEED=42 /opt/python/cp27-cp27m/bin/python setup.py test --parallel=4)
```
Comment (bsipocz) [inline .circleci/config.yml]: maybe also try to remove my none working magic, too?

```suggestion
          command: PYTHONHASHSEED=42 /opt/python/cp27-cp27m/bin/python setup.py test --offline --no-git --parallel=4 
```

**Comment by bsipocz** [comment]
Oh, the magic is gone :(

**Comment by pllim** [comment]
Add the old workaround back? :grimacing: :crossed_fingers: :crystal_ball:

**Comment by bsipocz** [comment]
still no good. maybe try your magic on the second one (i suppose the failure in https://circleci.com/gh/astropy/astropy/17606?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link shows that it never worked in the first place?)

**Comment by astrofrog** [comment]
@bsipocz - it doesn't work. I'm trying something else

**Comment by astrofrog** [comment]
So this now works. I think the exit code of the original command must have been zero or something, and adding ``--offline --no-git`` make it actually error properly, falling back on the second command.

**Comment by bsipocz** [comment]
Thank you @astrofrog!

**Comment by bsipocz** [comment]
No need to wait for travis/appveyor here, so I'm merging this right away ;)

**Pull request merged**